scripts/packages/pyheron/README.txt

```diff
@@ -0,0 +1,2 @@
+PyHeron is the python API for Heron, the distributed streaming engine. The API is
+based on the Storm python API provided by Parsely.
```

Comment:
Can you provide a link to Parsely for reference. Also should we change to README.md to be consistent with other readmes in the repo?